src/main/java/org/intellij/sdk/codesync/codeSyncSetup/CodeSyncSetup.java

```diff
@@ -138,6 +140,7 @@ public void run(@NotNull ProgressIndicator progressIndicator) {
 
                 // Finished
                 codeSyncProgressIndicator.setMileStone(InitRepoMilestones.END);
+                pluginState.syncInProcess = false;
```

Comment:
@Basit-qc I discussed with Saleem and we have to make changes in this method only and we can't change status in any other place. If code is not reaching at line 143 then we must figure out what unhandled exceptions are causing it and handle them so it must reach line 143.

Reply:
Sounds good 👍